pubspec.yaml

```diff
@@ -8,5 +8,6 @@ environment:
 
 dev_dependencies:
   async: ^2.5.0
+  fake_async: ^1.3.0
```

Comment:
I don't really *trust* fake_async, but for simple things, it's probably fine.
(Have seen too many people misuse it it internal tests, because of zones.)

Reply:
I think our choices are either do this, or bump up the delays so that we can tolerate 15ms of error.

I do think this is the right type of use case for `fake_async`.